scripts/deploy.ts

```diff
@@ -383,16 +398,75 @@ function buildCommonPost(buildObj: BuildObj) {
 
   if (buildObj.rsyncLocation != null) {
     mylog(`\n\nUploading to rsyncLocation ${buildObj.rsyncLocation}`)
-    mylog('***********************\n')
+    mylog('***********************************************************************\n')
 
     // const rsyncDir = join(buildObj.rsyncLocation, repoBranch, platformType, String(buildNum))
 
     const datePrefix = new Date().toISOString().slice(2, 19).replace(/:/gi, '').replace(/-/gi, '')
-    const rsyncFile = escapePath(`${datePrefix}--${productNameClean}--${platformType}--${repoBranch}--${buildNum}--${guiHash.slice(0, 8)}.ipa`)
+    const fileSuffixArr = buildObj.ipaFile.split('.')
+    const fileSuffix = fileSuffixArr[fileSuffixArr.length - 1]
```

Comment:
Propose using the word "extension" or "ext" instead of "suffix". You can also shorten this to one line.

```
    const [fileExtension] = buildObj.ipaFile.split('.').reverse()
```